c/tskit/tables.c

```diff
@@ -5622,8 +5623,21 @@ tsk_table_collection_check_integrity(tsk_table_collection_t *self, tsk_flags_t o
         goto out;
     }
 
+    /* Individuals */
+    for (j = 0; j < self->individuals.location_offset[self->individuals.num_rows]; j++) {
+        if (! isfinite(self->individuals.location[j])) {
+            ret = TSK_ERR_SPATIAL_LOCATION_NONFINITE;
```

Comment:
I do not think we should be throwing an error if spatial locations are not finite. For tskit, the spatial locations are just metadata, really, and we don't have any business enforcing anything there. (For instance, maybe someone wants to use inf as a special value...)

Reply:
I defer to https://github.com/tskit-dev/tskit/issues/293#issuecomment-520066132

Reply:
Whoops, sorry, missed that in the skim of this thread. =)

Ok, fine. We don't do anything with locations at the moment, but I can imagine using them in the future...

Reply:
Yeah, the whole point of keeping locations as a column rather just metadata was that we might want to do calculations on them at some point, so we should enforce that they make sense here.